### ISSUE TITLE
fails to build with -fno-common or gcc-10

### ISSUE THREAD

**Issue by johu** [baskerville/bspwm#1083]
```
x86_64-pc-linux-gnu-gcc -O2 -pipe -march=native -fno-common -std=c99 -pedantic -Wall -Wextra -DJSMN_STRICT -D_POSIX_C_SOURCE=200809L -DVERSION=\"0.9.9\"  -c -o rule.o src/rule.c
x86_64-pc-linux-gnu-gcc -O2 -pipe -march=native -fno-common -std=c99 -pedantic -Wall -Wextra -DJSMN_STRICT -D_POSIX_C_SOURCE=200809L -DVERSION=\"0.9.9\"  -c -o ewmh.o src/ewmh.c
x86_64-pc-linux-gnu-gcc -O2 -pipe -march=native -fno-common -std=c99 -pedantic -Wall -Wextra -DJSMN_STRICT -D_POSIX_C_SOURCE=200809L -DVERSION=\"0.9.9\"  -c -o subscribe.o src/subscribe.c
x86_64-pc-linux-gnu-gcc -Wl,-O1 -Wl,--as-needed  bspwm.o helpers.o geometry.o jsmn.o settings.o monitor.o desktop.o tree.o stack.o history.o events.o pointer.o window.o messages.o parse.o query.o restore.o rule.o ewmh.o subscribe.o  -Wl,-O1 -Wl,--as-needed -lm -lxcb -lxcb-util -lxcb-keysyms -lxcb-icccm -lxcb-ewmh -lxcb-randr -lxcb-xinerama -lxcb-shape -o bspwm
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: helpers.o:(.bss+0x1a8): multiple definition of `screen'; bspwm.o:(.bss+0x1a8): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: helpers.o:(.bss+0x0): multiple definition of `randr'; bspwm.o:(.bss+0x0): first defined here
```

Reported downstream, please see https://bugs.gentoo.org/706676 for further details.

**Comment by llchan** [comment]
I think it should be straightforward to fix this with some `extern` declarations.  I'll look into it when I have some spare time, maybe over the weekend.

**Comment by baskerville** [comment]
Fixed by 6c39ab08233d00feb5ad0b5e6b7ba954dc1b578a.
